### ISSUE TITLE
Error while reading junit file

### ISSUE THREAD

**Issue by ghost** [inception-health/otel-upload-test-artifact-action#6]
Debug:
```bash
Run inception-health/otel-upload-test-artifact-action@v1
  
Verify Job<Unit Test> and Step<Run tests>
Trace Test file
##[debug]matchDirectories 'false'
##[debug]followSymbolicLinks 'true'
##[debug]implicitDescendants 'true'
##[debug]matchDirectories 'false'
##[debug]omitBrokenSymbolicLinks 'true'
##[debug]Search path '/home/runner/work/service-template/service-template/tests-junit.xml'
##[debug]Tracing file: /home/runner/work/service-template/service-template/tests-junit.xml
##[debug]Tracing TestSuites<Undefined>
##[debug]Tracing TestSuite<github.com/my-org/service-template/cmd/app>
##[debug]Traced 1 Files
Error: TypeError: Cannot read properties of undefined (reading 'map')
```

Junit Sample:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/cmd/app" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/cmd/app/docs" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
	<testsuite tests="3" failures="0" time="0.008000" name="github.com/my-org/service-template/internal/nodetemplate" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New/succesfully_creates_client" time="0.000000"></testcase>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New/unsuccesfully_creates_client_with_missing_host" time="0.000000"></testcase>
		<testcase classname="github.com/my-org/service-template/internal/nodetemplate" name="Test_New" time="0.000000"></testcase>
	</testsuite>
	<testsuite tests="0" failures="0" time="0.000000" name="github.com/my-org/service-template/internal/nodetemplate/models" timestamp="2022-06-14T22:02:21Z">
		<properties>
			<property name="go.version" value="go1.18.3 linux/amd64"></property>
		</properties>
	</testsuite>
</testsuites>
```

**Comment by v1v** [comment]
I've faced a similar issue with https://github.com/mdelapenya/junit2otlp/blob/main/TEST-sample.xml, and it's also coming from golang.

**Comment by v1v** [comment]
Similar issue while using `Node.js` TAP results transformed to `JUnit`:
- https://github.com/elastic/apm-agent-nodejs/runs/7210518877?check_suite_focus=true
- https://www.npmjs.com/package/tap-junit

**Comment by v1v** [comment]
If you'd like to try with my fork, I put the fix from #9 

```
        uses: v1v/otel-upload-test-artifact-action@v2
        if: always()
        with:
```
